refs/x-compile.sh

```diff
@@ -126,27 +179,29 @@ Build_CT-NG() {
 	# XXX: Building the PW2 TC against glibc 2.19, on the other hand, results in working binaries... WTF?! (5.3 2016.03 snapshot)
 	# NOTE: Joy? The issue appears to be fixed in Linaro GCC 5.3 2016.04 snapshot! :)
 
-	# FIXME: I'm seriously considering adding proper support for binutil's native handling of the LTO plugin, via lib/bfd-plugins, instead of relying on the GCC wrappers...
+	# XXX: I'm seriously considering adding proper support for binutil's native handling of the LTO plugin, via lib/bfd-plugins, instead of relying on the GCC wrappers...
 	#        cd ${HOME}/x-tools/arm-kobo-linux-gnueabihf
 	#        mkdir -p lib/bfd-plugins
 	#        cd lib/bfd-plugins
 	#        ln -sf ../../libexec/gcc/arm-kobo-linux-gnueabihf/7.3.1/liblto_plugin.so.0.0.0 liblto_plugin.so
+	# NOTE: This *should* be automatically taken care of on ct-ng 1.24 ;).
 
-	./bootstrap
-	./configure --prefix=${HOME}/Kindle/CrossTool
-	make
-	make install
-	export PATH="${PATH}:${HOME}/Kindle/CrossTool/bin"
+	# NOTE: Performance appears to have gone *noticeably* down between GCC Linaro 7.4 and ARM 8.3 (and even more sharply between ARM 8.3 & FSF 9.2)... :/
+	#       Possibly related: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=91598
+	#       TL;DR: We're not moving away from the final Linaro TC for now. Yay?
+	#              The good news is all the work involved in moving to ct-ng 1.24 is done, and building Linaro 7.4 also works there, FWIW.
 
-	cd ..
-	mkdir -p TC_Kindle
-	cd TC_Kindle
+	git clean -fxdq
+	./bootstrap
+	./configure --enable-local
+	make -j8
```

Comment:
Probably better to use `nproc`?

Reply:
Nah, there's almost nothing to be built (kconfig, basically), and I'm probably the only one using that version of the script somewhere ;).

Reply:
The `gen-tc.sh` version is a plain `make`, no jobs

Reply:
So how does it use up all my CPU? :-P

Reply:
Although, yeah, `nproc` is probably better than a random hardcoded number :D.

Reply:
Indeed, because you are programming not just because you want to accomplish some specific task, but, presumably, also because you are leaving a work of art to be observed and learned from by the young generation. It is better for them to see `$(nproc)` than some hard-coded magic number, as you also agree :)

Reply:
The actual TC build process re-computes that on its own later, that make basically just builds ct-ng itself ;).